web/src/pages/Environments/EnvironmentCard.tsx

```diff
@@ -0,0 +1,122 @@
+import {DownOutlined, RightOutlined} from '@ant-design/icons';
+import {Dropdown, Menu, Typography} from 'antd';
+import {useCallback, useState} from 'react';
+import * as T from '../../components/TestCard/TestCard.styled';
+import {IEnvironment, useLazyGetEnvironmentSecretListQuery} from '../../redux/apis/TraceTest.api';
+import EnvironmentsAnalytics from '../../services/Analytics/EnvironmentsAnalytics.service';
+import * as S from './Envs.styled';
+
+interface IProps {
+  openDialog: (mode: boolean) => void;
+  environment: IEnvironment;
+  setEnvironment: (mode: IEnvironment) => void;
+}
+
+export const EnvironmentCard = ({
+  setEnvironment,
+  openDialog,
+  environment: {name, id, description},
+}: IProps): React.ReactElement => {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const [loadResultList, {data: resultList = []}] = useLazyGetEnvironmentSecretListQuery();
+  const onCollapse = useCallback(async () => {
+    EnvironmentsAnalytics.onEnvironmentClick(id);
+
+    if (resultList.length > 0) {
+      setIsCollapsed(true);
+      return;
+    }
+    await loadResultList({environmentId: id, take: 5});
```

Comment:
I think we can implement a skeleton loader similar to what @jorgeepc implemented for the new TestCard

Reply:
✅